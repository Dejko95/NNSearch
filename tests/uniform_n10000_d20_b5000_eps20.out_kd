Max visited: 10014
Avg visited: 9200.5105
