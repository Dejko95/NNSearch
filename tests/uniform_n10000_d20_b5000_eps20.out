
Max visited: 11534
Avg visited: 5714.0809
Max error: 0.0
Avg err: 0.0
Precision: 100 %
